### ISSUE TITLE
Score should be deterministic

### ISSUE THREAD

**Issue by tomohisaota** [tarao/prisoners-switch#19]
It seems that score varies each time I run the program.
That variation make it hard to compare different implementation.

**Comment by keijak** [comment]
Have you tried setting a random seed in your local run yet? That may give you deterministic score but if an algorithm performs very differently with two different seeds, such determinism is hardly useful anyway.

**Comment by tarao** [comment]
I agree with @keijak.  Essentially, the performance of a strategy cannot be determined accurately by concrete examples, I think.  You need a probabilistic analysis to prove that one strategy defeats another one.

**Comment by tomohisaota** [comment]
I have tried setting rand seed to fixed value, but it did not help.
I am newbie to golang, so I may be doing it in a wrong way.

Now I see that you want to avoid overfitting to fixed random input.
But on the other hand, performance of a strategy cannot be determined accurately if identical logics give different score every time.
We cannot tell if one logic is better than the other, or just luckier,

**Comment by keijak** [comment]
Well, you can:-) That's why statistical analysis is a powerful tool.
With a large number of trials, you can tell consistently good strategies from consistently bad ones and sometimes-good-sometimes-bad ones.
